src/include/sof/audio/kpb.h

```diff
@@ -57,6 +57,11 @@
 #define KPB_ALLOCATION_STEP 0x100
 #define KPB_NO_OF_MEM_POOLS 3
 
+enum kpb_state {
+	KPB_BUFFERING = 0,
```

Comment:
Lets namespace these to follow enum name i.e. KPB_STATE_

Reply:
@lgirdwood, changes applied.